internal/core/bootstrap.go

```diff
@@ -57,7 +64,9 @@ func init() {
 
 	processes, err := process.Processes()
 	if err != nil {
-		sessionID = uuid.New().String()
+		if sessionID == "" {
+			sessionID = uuid.New().String()
+		}
```

Comment:
suggestion: put this in a `defer` check in an else block to the `cfg.SessionID != ""` less to maintain that way.

Reply:
I'm no longer pushing this change.